apps/ex_wire/lib/ex_wire/adapter/tcp/server.ex

```diff
@@ -267,6 +266,8 @@ defmodule ExWire.Adapter.TCP.Server do
         peer.remote_id
       )
 
+    Logger.debug("[Network] Generating EIP8 Handshake for #{peer.host}")
```

Comment:
I added this in here in place of the one that read, 

```elixir
 Logger.debug("[Network] Sending EIP8 Handshake to #{remote_addr}")
```

in `EIP8.wrap_eip_8` function. 

I'm not sure how much value these are really providing. I'm in favor of removing this if others agree that there isn't much value in this.

Reply:
Makes sense. I think we'll have to think about how to do that holistically, so that we can turn all logging on or off based on flags provided.